packages/grid/_modules_/grid/components/columnHeaders/GridColumnHeaderItem.tsx

```diff
@@ -140,15 +144,12 @@ export const GridColumnHeaderItem = ({
 
   const columnTitleIconButtons = (
     <React.Fragment>
-      <GridColumnHeaderSortIcon
-        direction={sortDirection}
-        index={sortIndex}
-        hide={column.hideSortIcons}
-      />
       <ColumnHeaderFilterIcon counter={filterItemsCounter} />
+      {column.sortable && !column.hideSortIcons && (
```

Comment:
I had to invert the filter icon and the sort icon or it gonna have an empty space when the column is unsorted, but there's an active filter.

![image](https://user-images.githubusercontent.com/42154031/116129315-5a022400-a6a0-11eb-8975-1673f377e5e3.png)